gradle/maven.gradle

```diff
@@ -39,7 +39,7 @@ plugins.withId('maven') {
                     }
                 }
 
-                pom.artifactId = project.path.replace(':', '')
+                pom.artifactId = project.path.replaceFirst(':', '').replace(':', '-')
```

Comment:
A comment with an example'd be helpful here to understand why this is needed.

Reply:
`":serializers:avro".replaceFirst(":","")` would become `"serializers:avro"` and then `replace(":", "-")` would make it `"serializers-avro"`